test/arrays/slices/sliceAssoc.chpl

```diff
@@ -0,0 +1,20 @@
+const ProgModels = {"C", "Chapel", "MPI", "OpenMP"};
+const CoolLanguages = {"Chapel"};
+
+var LinesOfCode: [ProgModels] int;
+
+LinesOfCode["C"] = 10000;
+LinesOfCode["Chapel"] = 1000;
+LinesOfCode["MPI"] = 12000;
+LinesOfCode["OpenMP"] = 11000;
```

Comment:
"Just have a look at this Very Scientific Chart I made comparing programming models"